packages/typescript/src/components/DeclarationJSDoc.tsx

```diff
@@ -0,0 +1,18 @@
+import { Show } from "@alloy-js/core";
+import { JSDoc } from "./JSDoc.jsx";
+import { JSDocParagraph } from "./JSDocParagraph.jsx";
+
+export interface DeclarationJSDocProps {
+  doc?: string;
```

Comment:
I feel like this doc should support multiple paragraphs. Maybe take `string | string[]`?